xarray/tutorial.py

```diff
@@ -39,7 +39,7 @@ def _construct_cache_dir(path):
 file_formats = {
     "air_temperature": 3,
     "air_temperature_gradient": 4,
-    "ASE_ice_velocity":3,
+    "ASE_ice_velocity": 3,
```

Comment:
```suggestion
    "ASE_ice_velocity": 4,
```

`ncdump -k ASE_ice_velocity` says "netCDF-4" so we need `4` instead of `3`. `ncdump -k air_temperature` says "classic" which is a netCDF-3 file.